apps/src/maze/bee.js

```diff
@@ -1,475 +1,468 @@
-var utils = require('../utils');
-var mazeMsg = require('./locale');
-var BeeCell = require('./beeCell');
-var TestResults = require('../constants.js').TestResults;
-var TerminationValue = require('../constants.js').BeeTerminationValue;
-
-var UNLIMITED_HONEY = -99;
-var UNLIMITED_NECTAR = 99;
-
-var EMPTY_HONEY = -98; // Hive with 0 honey
-var EMPTY_NECTAR = 98; // flower with 0 honey
-
-var Bee = function (maze, studioApp, config) {
-  this.maze_ = maze;
-  this.studioApp_ = studioApp;
-  this.skin_ = config.skin;
-  this.defaultFlowerColor_ = (config.level.flowerType === 'redWithNectar' ?
-    'red' : 'purple');
-  if (this.defaultFlowerColor_ === 'purple' &&
-    config.level.flowerType !== 'purpleNectarHidden') {
-    throw new Error('bad flowerType for Bee: ' + config.level.flowerType);
```

Comment:
Ah, yeah, and unfortunately our standard of extra-indenting continuations is also not supported by any code beautifier that I've tried.